client/src/components/FoodSeeker/SearchResults/layouts/Desktop.js

```diff
@@ -87,7 +87,7 @@ const DesktopLayout = ({ filters, list, map }) => {
             left: isListPanelOpen ? 0 : "-524px",
             top: "120px",
             height: "100%",
-            zIndex: "1",
+            zIndex: "3",
```

Comment:
Keep as numeric value, only use string defaults such as `tooltip` or `modal` for preset arbitrary number.